test/unit-tests/kitmanager.test.ts

```diff
@@ -23,4 +25,28 @@ suite('Kits test', async () => {
       'VSCode Kit 2',
     ]);
   });
+
+  test('Test load env vars from shell script', async() => {
+    const fname_extension = process.platform == 'win32' ? 'bat' : 'sh';
+    const fname = `cmake-kit-test-${Math.random().toString()}.${fname_extension}`;
+    const script_path = path.join(paths.tmpDir, fname);
+    // generate a file with test batch / shell script that sets two env vars
+    if (process.platform == 'win32')
+      await fs.writeFile(script_path, `set "TESTVAR12=abc"\r\nset "TESTVAR13=cde"`);
+    else
+      await fs.writeFile(script_path, `export "TESTVAR12=abc"\r\nexport "TESTVAR13=cde"`);
```

Comment:
for non-Windows, don't add a `\r` with the newline

Reply:
Thats right